src/kfactory/kcell.py

```diff
@@ -2595,6 +2599,8 @@ def cell(
         basename: str | None = None,
         drop_params: list[str] = ["self", "cls"],
         register_factory: bool = True,
+        info: dict[str, MetaData] | None = None,
+        post_process: Iterable[Callable[[Any], None]] = [],
```

Comment:
this is not always true is it? what happens in case of add_pins? you get a new kcell, right?

Reply:
add pins never returns a new Cell, but modifies the Pcell in place